src/hooks/use_notification.rs

```diff
@@ -11,6 +11,19 @@ pub struct NotificationItem {
     pub handle: NotificationHandle,
 }
 
+impl Default for NotificationItem {
```

Comment:
Would be much easier to `#[derive(Default)]`, from what I can see you are using default values for all properties anyway. `NotificationType` can also derive Default but remember to ad `#[default]` to the `None` variant.